hologram/__init__.py

```diff
@@ -956,6 +956,6 @@ def _get_field_type_name(field_type: Any) -> str:
     def validate(cls, data: Any):
         schema = _validate_schema(cls)
         validator = jsonschema.Draft7Validator(schema)
-        error = jsonschema.exceptions.best_match(validator.iter_errors(data))
+        error = next(validator.iter_errors(data), None)
```

Comment:
I'm not sure why hologram is upset about this, but I think you want `next(iter(validator.iter_errors(data)), None)` here.